### PR TITLE
add specific backbone view doc

### PR DESCRIPTION
**Pull request by nicolasThal**
@juchi

### CODE REVIEW

File: en/developer_guide/specific_backbone_view.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+In some case, during the integration of the Open Orchestra Backoffice, the generic view used for the
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: some cases

File: en/developer_guide/specific_backbone_view.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+edition page is not enough. There are some new behaviour to add to the form, such as a specific event
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: i would rephrase : "is not enough" -> "will not suit your needs" or even "you may want to use a custom view"

File: en/developer_guide/specific_backbone_view.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+when you edit a field.
+
+There are two simple solution to avoid :
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: solutions

File: en/developer_guide/specific_backbone_view.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+ - Add some javascript in the form template
+ - Add a global listener to the javascript project
+
+Both of these solution are to avoid because :
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: solutions

File: en/developer_guide/specific_backbone_view.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+ - They force a link between the templating and the javascript
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: maybe "They couple the tpl and JS" ?

File: en/developer_guide/specific_backbone_view.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+ - The may be checked each time the dom is modified
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: DOM is capitalized

File: en/developer_guide/specific_backbone_view.rst [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+To prevent these usage, Open Orchestra provide a way to modify the view used in Backbone.
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst:26]: usages, provides
i would rephrase the end : the Backbone view used by Orchestra

File: en/developer_guide/specific_backbone_view.rst [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+Usage
+-----
+
+As all the crud are based on an ``entityType``, you will have the possibility to add a view for the
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: CRUD is capitalized

File: en/developer_guide/specific_backbone_view.rst [hunk 9]
```diff
@@ -0,0 +1,37 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+In some case, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page is not enough. There are some new behaviour to add to the form, such as a specific event
+when you edit a field.
```
Comment (juchi) [inline en/developer_guide/specific_backbone_view.rst]: After this, I would quickly explain that it's possible to fully manage the view from the listing page, but if the dev wants to use the orchestra listing view and yet change the edit action, then here's the way to go.

**Comment by ngilain** [comment]
@nicolasThal : I find this doc ambiguous on the problem it's trying to solve: backbone view vs edit form template.
As we talked earlier, if you want to present a specific edit form for a specific content type, you only have to set a specific template when editing the content type. Then, all contents from that content type will be edited with that template. That way you can add a specific behavior to a form.

But that doc is not talking about the form, it's talking about it's container. Maybe that point should be highlited to avoid confusion.

**Comment by nicolasThal** [comment]
all right, I will correct the doc this way :)

**Comment by nicolasThal** [comment]
@juchi @itkg-ngilain update done

File: en/developer_guide/specific_backbone_view.rst [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+There are two simple solutions to avoid :
+
+ - Add some javascript in the form template
+ - Add a global listener to the javascript project
+
+Both of these solutions are to avoid because :
+
+ - They couple the template and the javascript
+ - The may be checked each time the DOM is modified
```
Comment (ngilain) [inline en/developer_guide/specific_backbone_view.rst]: They

File: en/developer_guide/specific_backbone_view.rst [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+To prevent these usages, Open Orchestra provides a way to modify the Backbone view used by default.
+
+Usage
+-----
+
+As all the CRUD are based on an ``entityType``, you will have the possibility to add a view for the
+``edit`` and ``add`` action on this ``entityType``.
```
Comment (ngilain) [inline en/developer_guide/specific_backbone_view.rst:32]: How to do if the entity type is Content, but the modification target is only one Content Type, news for instance ?

File: en/developer_guide/specific_backbone_view.rst [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+Context
+-------
+
+As all the Backoffice of Open Orchestra is based on Backbone view system, all the specific event
+linked to a view are linked to the view directly.
+
+In some cases, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page will not suit your needs. There are some new behaviour to add to the form, such as a
+specific event when you edit a field.
```
Comment (ngilain) [inline en/developer_guide/specific_backbone_view.rst:14]: I find the general explanation better than the first version, but I'll add a little explanation on how an edit form page is built: two templates are involved. The first is the backbone view for the 'container' aspect (including events binding), the second is the template form describing the rendering of the fields.
The good practice you're proposing is to bind the events on the backbone view rather than the form template.

File: en/developer_guide/specific_backbone_view.rst (hunk 11 above)
Comment (nicolasThal) [inline en/developer_guide/specific_backbone_view.rst:32]: In this case the content type is set in the entityType name : for a news, it will be contents_news.

I will add it in the documentation

File: en/developer_guide/specific_backbone_view.rst (hunk 12 above)
Comment (nicolasThal) [inline en/developer_guide/specific_backbone_view.rst:14]: I think that should be a full doc on how the crud works.

**Comment by nicolasThal** [comment]
@itkg-ngilain update done

File: en/developer_guide/specific_backbone_view.rst (hunk 12 above)
Comment (ngilain) [inline en/developer_guide/specific_backbone_view.rst:14]: Maybe you could rephrase the intro sentence because it says something redundant : "the specific event linked to a view are linked to the view" which won't be helpfull for someone discovering Open Orchestra :)

Maybe that intro could tell that an edit form is composed of two parts, the backbone view and the edit form template, and that the form events should be bind in the backbone view, not the template form.

**Comment by nicolasThal** [comment]
@itkg-ngilain update done

File: en/developer_guide/specific_backbone_view.rst [hunk 13]
```diff
@@ -0,0 +1,51 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+In the Open Orchestra Backoffice, to display an edition form, there are two part :
```
Comment (ngilain) [inline en/developer_guide/specific_backbone_view.rst]: two parts

**Comment by ngilain** [comment]
@nicolasThal : I'm ok with that version :) :+1: with the little spellchecking
